### PR TITLE
try SymPy.jl

### PR DESCRIPTION
**Pull request by anandijain**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -215,14 +220,16 @@ function test_integrals(; kw...)
...
     end
     return n
 end
 
-@testset "integral" begin 
-    n = test_integrals(;symbolic=false, verbose=false, homotopy=true, num_steps=2, num_trials=10)
-    @test n==11
+@testset "integral" begin
+    n = test_integrals(; symbolic=false, verbose=false, homotopy=true, num_steps=2, num_trials=10)
+    @test n == 10
```
Comment (anandijain) [inline test/runtests.jl:234]: if tests pass, this is really the only thing of note. note sure why one more integral works now

**Comment by anandijain** [comment]
@ChrisRackauckas squash this shit

**Comment by shahriariravanian** [comment]
It is better to split the SymPy dependency. It is not needed for actual use and is just for internal testing and development. Let's make a separate repository, say SymbolicNumericIntegrationTesting.jl or such, and move some of the obscure code there.

**Comment by anandijain** [comment]
I wouldn't do that now that its sorted out

**Comment by anandijain** [comment]
its only a test dependency now so it shouldn't be that much of an issue

**Comment by anandijain** [comment]
btw next time you should squash and merge, otherwise the tree is ugly

**Comment by ChrisRackauckas** [comment]
Awesome @anandijain !

**Pull request merged**